packages/@glimmer/runtime/lib/vm/attributes/dynamic.ts

```diff
@@ -188,6 +192,22 @@ export class OptionSelectedDynamicAttribute extends DefaultDynamicProperty {
   }
 }
 
+export class BooleanDynamicAttribute extends DefaultDynamicProperty {
+  set(dom: ElementBuilder, value: Opaque, env: Environment): void {
+    let normalizedValue = normalizeValue(value);
+    super.set(dom, normalizedValue, env);
+  }
+
+  update(value: Opaque, env: Environment): void {
+    let normalizedValue = normalizeValue(value);
+    super.update(normalizedValue, env);
+  }
+}
+
+function isBooleanAttribute(tagName: string, attribute: string) {
+  return tagName === 'BUTTON' && attribute === 'disabled';
```

Comment:
Why is this narrowed to `BUTTON`?

Reply:
Just for clarity in this POC i would say, the complete list of attributes is 
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes, and disabled attribute is applicable to more tags, like i said this is just an approach which needs to be updated according to the list of Boolean attributes. I’ll update the list if this approach seems fine to you ☺️